modules/javafx.controls/src/main/java/javafx/scene/control/TableView.java

```diff
@@ -395,6 +395,92 @@
         }
     };
 
+    /**
+     * A policy that tries to adjust other columns in order to fit the table width.
+     * <p>
+     * During UI adjustment, proportionately resizes all columns.
+     * <p>
+     * When column constraints make it impossible to fit all the columns into the allowed area,
+     * the columns are either clipped, or an empty space appears.  This policy disables the horizontal
+     * scroll bar.
+     *
+     * @since 20
+     */
+    public static final Callback<TableView.ResizeFeatures, Boolean> CONSTRAINED_RESIZE_POLICY_ALL_COLUMNS =
+        ConstrainedColumnResize.forTable(ConstrainedColumnResize.ResizeMode.AUTO_RESIZE_ALL_COLUMNS);
+
+    /**
+     * A policy that tries to adjust last column in order to fit the table width.
+     * <p>
+     * During UI adjustment, resizes the last column only.
+     * <p>
+     * When column constraints make it impossible to fit all the columns into the allowed area,
+     * the columns are either clipped, or an empty space appears.  This policy disables the horizontal
+     * scroll bar.
+     *
+     * @since 20
+     */
+    public static final Callback<TableView.ResizeFeatures, Boolean> CONSTRAINED_RESIZE_POLICY_LAST_COLUMN =
+        ConstrainedColumnResize.forTable(ConstrainedColumnResize.ResizeMode.AUTO_RESIZE_LAST_COLUMN);
+
+    /**
+     * A policy adjusts the next column in the opposite way in order to fit the table width.
+     * <p>
+     * During UI adjustment, resizes the next column the opposite way.
+     * <p>
+     * When column constraints make it impossible to fit all the columns into the allowed area,
+     * the columns are either clipped, or an empty space appears.  This policy disables the horizontal
+     * scroll bar.
+     *
+     * @since 20
+     */
+    public static final Callback<TableView.ResizeFeatures, Boolean> CONSTRAINED_RESIZE_POLICY_NEXT_COLUMN =
+        ConstrainedColumnResize.forTable(ConstrainedColumnResize.ResizeMode.AUTO_RESIZE_NEXT_COLUMN);
+
+    /**
+     * A policy that tries to adjust subsequent columns in order to fit the table width.
+     * <p>
+     * During UI adjustment, resizes subsequent columns to preserve the total width.
```

Comment:
Maybe say `proportionately resizes...` to match the language in `CONSTRAINED_RESIZE_POLICY_ALL_COLUMNS`?